requirements.txt

```diff
@@ -3,6 +3,7 @@ grpcio==1.1.3
 grpcio-tools==1.1.3
 kubernetes==1.0.0
 oslo.log==3.28.0
+oslo.messaging==5.28.0
```

Comment:
The Oslo cfg module generates an error when registering configuration options if oslo.messaging is not installed.